### PR TITLE
Update NORD subset

### PR DESCRIPTION
**Pull request by matentzn**

### CODE REVIEW

File: src/sparql/qc/mondo/qc-illegal-prefix-on-xref.sparql [hunk 1]
```diff
@@ -49,6 +49,7 @@ WHERE
       "NCIT",
       "NDFRT",
       "NIFSTD",
+      "NORD",
```
Comment (joeflack4) [inline src/sparql/qc/mondo/qc-illegal-prefix-on-xref.sparql:52]: Hmm, I see these prefixes are supposedly illegal, but not sure why.

Comment (sabrinatoro) [reply src/sparql/qc/mondo/qc-illegal-prefix-on-xref.sparql:52]: @joeflack4 I just created these.

Comment (matentzn) [reply src/sparql/qc/mondo/qc-illegal-prefix-on-xref.sparql:52]: These are the legal ones

**Review by sabrinatoro — APPROVED** [review]
@matentzn 
this PR can be merged. Since it looks like Joe is also reviewing, I will not merge. 

Could you please confirm that "MONDO:NORD" as a source is what is expected? Thank you!

**Comment by matentzn** [comment]
Yes, for now! We have not discussed group provenance in mondo but this is a good placeholder. Thank you!

File: src/sparql/qc/mondo/qc-illegal-prefix-on-xref.sparql (hunk 1 above)
Comment (joeflack4) [reply src/sparql/qc/mondo/qc-illegal-prefix-on-xref.sparql:52]: Ah I see now the `    FILTER( STRBEFORE(str(?source),":") not in (`

**Review by joeflack4 — APPROVED** [review]

**Comment by joeflack4** [comment]
Sorry meant to approve earlier.

**Comment by sabrinatoro** [comment]
@matentzn 
I am confirming that the number of NORD x-refs that were added to this PR is 974.
If that number is what you expected, please feel free to merge the PR. 
Thank you!

**Comment by matentzn** [comment]
Great you checked this; we almost destroyed the rare disease subset :D 

This is now superceded: #7016

**Pull request closed without merging**